blackbox/docs/general/ddl/create-view.rst

```diff
@@ -0,0 +1,41 @@
+==============
+Creating Views
+==============
+
+.. rubric:: Table of Contents
+
+.. contents::
+    :local:
+
+Introduction
```

Comment:
looks a bit strange to me to have 1 sub-topic in this document and a TOC. would remove the TOC and sub-topic completely.